optaplanner-persistence/optaplanner-persistence-jsonb/src/test/java/org/optaplanner/persistence/jsonb/api/score/buildin/bendablelong/BendableLongScoreJsonbAdapterTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jsonb.api.score.buildin.bendablelong;
+
+import javax.json.bind.annotation.JsonbTypeAdapter;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.buildin.bendablelong.BendableLongScore;
+import org.optaplanner.persistence.jsonb.api.score.AbstractScoreJsonbAdapterTest;
+
+public class BendableLongScoreJsonbAdapterTest extends AbstractScoreJsonbAdapterTest {
+
+    @Test
+    public void serializeAndDeserialize() {
+        assertSerializeAndDeserialize(null, new TestBendableLongScoreWrapper(null));
+
+        BendableLongScore score = BendableLongScore.of(new long[]{1000L, 200L}, new long[]{34L});
+        assertSerializeAndDeserialize(score, new TestBendableLongScoreWrapper(score));
+
```

Comment:
Some of the test classes don't have the blank lines in here. I don't mind if you want to keep them - but in that case, be consistent, please.

Reply:
Removing them for consistency with other modules.